### ISSUE TITLE
[BUG] Translation domain var not present - breaking ReCaptcha

### ISSUE THREAD

**Issue by Maelstromeous** [bolt/boltforms#131]
I enabled the ReCaptcha in my config, but unfortunately the twig `translation_domain` variable is missing, therefore breaking the template.

![translation_domain](https://cloud.githubusercontent.com/assets/1776058/21010696/70ddb796-bd45-11e6-8225-d7f41f856f4f.png)

On that line:

```
<label for="form_message" class="required">{{ recaptcha.label|trans({}, translation_domain) }}</label>
```

**Comment by frostymarvelous** [comment]
Any feedback on this?

**Comment by SahAssar** [comment]
Looks to be fixed in the 4.0 branch, @rossriley Any ideas when you plan to tag?

**Comment by rossriley** [comment]
Very soon, just one minor bug that's turned up and we should be good to go.

Closing this since it's fixed in 4.0.
